src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * Math.pow(radius, 2);
+    }
```

Comment:
The `calculateArea()` method correctly calculates the area of the circle, but consider extracting the calculation `Math.PI * Math.pow(radius, 2)` into a separate constant or method for better readability and to avoid magic numbers.